.changeset/tall-parents-deny.md

```diff
@@ -0,0 +1,20 @@
+---
+'@backstage/plugin-search-react': minor
+---
+
+`SearchContextProviderForStorybook` and `SearchApiProviderForStorybook` has been deleted. New mock implementation of the `SearchApi` introduced. If you need to mock the api we recommend you to do the following:
```

Comment:
Let's highlight this with `**BREAKING**: `, since it was shipped in 1.1 right?

Reply:
it was exported in 0.1.0, see https://github.com/backstage/backstage/blob/master/plugins/search-react/CHANGELOG.md#010